invenio_app_ils/records/jsonresolver/item.py

```diff
@@ -11,8 +11,9 @@
 from invenio_circulation.api import get_loan_for_item
 
 
-@jsonresolver.route('/api/circulation/items/<pid_value>/loan',
-                    host='ils.mydomain.org')
+@jsonresolver.route(
+    "/api/circulation/items/<pid_value>/loan", host="ils.mydomain.org"
```

Comment:
Probably the host should be taken from config?

Reply:
@zzacharo If we try to load it from `current_app` it fails because it is out of request context and if we load it like `from config import ...` we loose the ability to override. I will create a ticker for this so we can handle it properly.

Reply:
Please link the ticket in this PR, then I think we can approve and merge :)